### PR TITLE
Add TestRun.start function. Bump version

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
/cc @inikulin 
We need this to synchronize LegacyTestRun with [this](https://github.com/DevExpress/testcafe/pull/744) changes in TestRun

### CODE REVIEW

File: src/test-run/index.js [hunk 1]
```diff
@@ -120,6 +120,9 @@ export default class LegacyTestRun extends Session {
 
         ctx.redirect(this.browserConnection.idleUrl);
     }
+
+    async start () {
```
Comment (inikulin) [inline src/test-run/index.js:124]: Add comment here please why we need it

**Comment by inikulin** [comment]
\r-

File: src/test-run/index.js (hunk 1 above)
Comment (AlexanderMoskovkin) [reply src/test-run/index.js:124]: `// NOTE: LegacyTestRun should have the same API with TestRun`
is it clear ?

Comment (inikulin) [reply src/test-run/index.js:124]: `// NOTE: required to maintain same API as TestRun. Just do nothing here.`

**Comment by AlexanderMoskovkin** [comment]
FPR (the comment is checked by @VasilyStrelyaev)

**Comment by inikulin** [comment]
lgtm

**Pull request merged**